ocs_ci/utility/deployment_openshift_logging.py

```
@@ -605,23 +625,37 @@ def install_logging(skip_resource_exists=False):
+        func=configmap_obj.get,
+        resource_name=constants.OBJECT_BUCKET_CLAIM,
+    ):
+        data = (cm or {}).get("data", {})
+        if data.get("BUCKET_NAME") and data.get("BUCKET_HOST"):
+            cm_dict = cm
+            break
+    if cm_dict is None:
+        raise ConfigMapDataNotAvailable(
+            f"ConfigMap {constants.OBJECT_BUCKET_CLAIM} exists but "
+            "BUCKET_NAME and BUCKET_HOST were not populated within the timeout"
+        )
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)